wiki/component-development.md

```diff
@@ -62,18 +62,18 @@ yarn build && npm pack
 
 This will create a `.tgz` file with the changes in your OUI directory. At this point you can move it anywhere.
 
-#### In Kibana:
+#### In OpenSearch Dashboards:
 
-Point the `package.json` file in Kibana to that file: `"@opensearch-project/oui": "/path/to/elastic-oui-xx.x.x.tgz"`. Then run the following commands at Kibana's root folder:
+Point the `package.json` file in OpenSearch Dashboards to that file: `"@opensearch-project/oui": "/path/to/elastic-oui-xx.x.x.tgz"`. Then run the following commands at OpenSearch Dashboards root folder:
```

Comment:
Should we change `/path/to/elastic-oui-xx.x.x.tgz` to something like `/path/to/oui-xx.x.x.tgz`?